### ISSUE TITLE
Some media keys cannot be re-assigned because Gnome Settings Daemon maps them statically

### ISSUE THREAD

**Issue by stan-janssen** [elementary/switchboard-plug-keyboard#376]
## What Happened

I'm on a desktop computer with one external monitor. I have a keyboard that has brightness up/down keys. I want to assign these keys to a script that I made that send DDC commands to my monitor to change the brightness. This script works fine if I assign, for example, `ctrl` + `brightness up` and `ctrl` + `brightness down`, but not the `brightness up` and `brightness down` keys on their own. The script simply does not run.

![brightness](https://user-images.githubusercontent.com/12174852/129094500-b841de7d-1a51-4739-a2fc-f809fc04b3bc.png)

## Expected Behavior
I'd like to be able to assign my `brightness up` and `brightness down` keys to a custom script.

## Steps to Reproduce

To make it simple, let's assign a shortcut key to the command `notify-send Hello "This is a test"`:

- Open System Settings > Keyboard > Shortcuts > Custom
- Add a custom shortcut for `notify-send Hello "This is a test"` with shortcut key `MonBrightnessUp`.
- Press the Brightness Up key; the script does not run.
- Change the shortcut to `ctrl` + `MonBrightnessUp`
- Press `ctrl` + `MonBrightnessUp`; the script runs and we see the notification.

![brightness-reproduce](https://user-images.githubusercontent.com/12174852/129094524-28b19590-030a-401d-91bf-51849cf95a32.png)

### Logs

I don't have any logs for this.

## Platform Information

ElementaryOS 6 stable. This problem was also present in elementaryOS 5.1.7 that I was using previously.

**Comment by sirsleon** [comment]
Can you assign any custom keyboard shortcuts? They don't seem to work for me at all.

**Comment by stan-janssen** [comment]
> Can you assign any custom keyboard shortcuts? They don't seem to work for me at all.

Yes, all other custom shortcuts work fine for me; see the description in the issue where setting `ctrl` + `Brightness Up` works fine. It seems to be that the OS is capturing the `Brightness Up` and `Brightness Down` keys and not propagate them to whatever system is handling these custom shortcuts.

**Comment by stan-janssen** [comment]
I've found the solution to this problem. Apparently, in Gnome Settings Daemon, there were bindings for the brightness keys named `screen-brightness-up-static` and `screen-brightness-down-static`. Once I removed the Brightness keys from these assignments, my custom keyboard shortcuts now work:

```
gsettings set org.gnome.settings-daemon.plugins.media-keys screen-brightness-down-static "['']"
gsettings set org.gnome.settings-daemon.plugins.media-keys screen-brightness-up-static "['']"
```

I've also verified that this solves the problem @sirsleon had in #378 where it was not possible to assign the `Tools` key; this binding key was already statically bound to `control-center-static`. To remove this static binding:

```
gsettings set org.gnome.settings-daemon.plugins.media-keys control-center-static "['']"
```

I think we should solve this problem by:

1. Disabling all static bindings for keys
2. Assigning some of these statically bound keys to correct elementaryOS actions (like `Tools` to `io.elementary.switchboard`)
